run_unittest_tests.js

```diff
@@ -0,0 +1,46 @@
+//
+// run_unittest_tests.js
```

Comment:
This could live in ./lib/tests maybe?  There's no super-obvious best home for it, but that doesn't seem too bad.